docs/version-history/development-release-series-91.rst

```diff
@@ -59,6 +59,11 @@ New Features:
   for other startd cron jobs; see :macro:`STARTD_CRON_<JobName>_CONDITION`.
   :jira:`667`
 
+- The :doc:`htcondor <../man-pages/htcondor>` CLI tool now automates the
```

Comment:
I might update this (if you manage to merge this soon) to instead use the `ref` I'm adding to the htcondor man-page rst in my PR

Reply:
Sounds good.